browser/brave_shields/ad_block_service_browsertest.cc

```diff
@@ -114,13 +114,17 @@ using brave_shields::features::kBraveAdblockScriptletDebugLogs;
 using brave_shields::features::kCosmeticFilteringJsPerformance;
 
 namespace {
-void WaitForSelectorBlocked(const content::ToRenderFrameHost& target,
-                            const std::string& selector) {
-  static constexpr char kTemplate[] =
-      R"(waitCSSSelector($1, 'display', 'none'))";
+void WaitForSelector(const content::ToRenderFrameHost& target,
+                     const std::string& selector,
+                     const bool must_be_blocked = true) {
+  static constexpr char kTemplate[] = R"(waitCSSSelector($1, 'display', '%s'))";
 
   ASSERT_TRUE(
-      EvalJs(target, content::JsReplace(kTemplate, selector)).ExtractBool());
+      EvalJs(target, content::JsReplace(
+                         base::StringPrintf(kTemplate,
```

Comment:
JsReplace supports more than one placeholder. I'd like to replace `StringPrintf` and just add $2 in the template